parsers/test_parser.py

```diff
@@ -0,0 +1,46 @@
+import time
+
+import arrow
+import click
+
+try:
+    from parsers.read_parser_config import parser_key_to_dict
+except Exception:
+    raise ValueError("couldn't do a local import. Please run from the"
+                     "`electricitymap` folder: python parsers/test_parser.py "
+                     "[OPTIONS] ZONE [DATA_TYPE]")
+
+
+@click.command()
+@click.argument('zone')
+@click.argument('data-type', default='production')
+@click.option('--target_datetime', default=None, show_default=True)
+def test_parser(zone, data_type, target_datetime):
+    if target_datetime:
+        target_datetime = arrow.get(target_datetime).datetime
```

Comment:
Isn't target_datetime a python datetime already?

Reply:
we parse `target_datetime`, which is a string that we got from the command line (using click), using the `arrow` library, then convert the `arrow` object to a `datetime`